front/src/applications/stdcmV2/types.ts

```diff
@@ -1,28 +1,36 @@
-import type {
-  RollingStockWithLiveries,
-  PostV2TimetableByIdStdcmApiResponse,
-} from 'common/api/generatedEditoastApi';
+import type { ManageTrainSchedulePathProperties } from 'applications/operationalStudies/types';
+import type { StdcmV2SuccessResponse } from 'applications/stdcm/types';
+import type { RollingStockWithLiveries } from 'common/api/generatedEditoastApi';
 import type { PathStep } from 'reducers/osrdconf/types';
 
-export type StdcmSimulationResult = {
-  createdAt: string;
-  input: {
-    origin?: PathStep | null;
-    departureDate?: string;
-    departureTime?: string;
-    destination?: PathStep | null;
-    pathSteps?: (PathStep | null)[];
-    consist?: {
-      tractionEngine: RollingStockWithLiveries | undefined;
-    };
+export type StdcmSimulationInputs = {
+  origin?: PathStep | null;
+  departureDate?: string;
+  departureTime?: string;
+  destination?: PathStep | null;
+  pathSteps?: (PathStep | null)[];
+  consist?: {
+    tractionEngine: RollingStockWithLiveries | undefined;
```

Comment:
Is there a reason why we don't use rollingStock for this property ? Is it because of the future implementation of the convoy ?

```suggestion
    tractionEngine?: RollingStockWithLiveries;
```

Reply:
Yes, the form is not complete.